src/jobflow_remote/jobs/jobcontroller.py

```diff
@@ -2562,7 +2568,12 @@ def get_flows_info(
         # Only use the full aggregation if more job details are needed.
         # The single flow document is enough for basic information
         if with_jobs_info:
```

Comment:
A weird usage of this would be to pass an empty list, which would then not follow what's mentioned in the docstring. Do you think it's worth changing the condition to something like:


```suggestion
        if with_jobs_info is not False:
```